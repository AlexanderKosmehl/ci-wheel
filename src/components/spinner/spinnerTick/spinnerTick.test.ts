import { describe, expect, it } from 'vitest';
import generateSpinnerTick from './spinnerTick';

describe('generateSpinnerTick', () => {
  const spinnerTick = generateSpinnerTick();

  it('generates component correctly', () => {
    expect(spinnerTick).toMatchSnapshot();
  });
});
