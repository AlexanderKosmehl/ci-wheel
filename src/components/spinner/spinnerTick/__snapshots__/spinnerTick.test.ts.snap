// Vitest Snapshot v1

exports[`generateSpinnerTick > generates component correctly 1`] = `
<div
  class="tickShadow"
>
  <div
    class="tick"
  />
</div>
`;
